filebeat/etc/beat.yml

```diff
@@ -1,192 +1,189 @@
-################### Filebeat Configuration Example #########################
+###################### Filebeat Configuration Example #########################
```

Comment:
I will do that in the PR adding the short ones, because I'm thinking the full and short should have a header mentioning the other option.